snapcraft/plugins/kernel.py

```diff
@@ -130,14 +130,16 @@ def schema(cls):
             'enum': ['gz'],
         }
 
+        return schema
+
+    @classmethod
+    def get_build_properties(cls):
         # Inform Snapcraft of the properties associated with building. If these
         # change in the YAML Snapcraft will consider the build step dirty.
-        schema['build-properties'].extend([
+        return kbuild.KBuildPlugin.get_build_properties() + [
```

Comment:
If I switch to super.get_build_properties() I get the following error:
'super' has no attribute 'get_build_properties'
Because it's a classmethod.